Makefile

```diff
@@ -462,7 +460,51 @@ homebrew-setup:
 # -------------------------------------------------
 # Nix
 # -------------------------------------------------
-#TODO: see flake.nix
+# See flake.nix top level comments for more information
+
+# Enter development environment with all dependencies installed
+shell:
+	$(eval USER_SHELL := $(shell finger ${USER} | grep 'Shell:*' | cut -f3 -d ":"))
```

Comment:
what is this thing?

Reply:
will add a comment, but it gets the user shell